### PR TITLE
updates to ajentvee, alleemac (alleemac bio unfinishied)

### PR DESCRIPTION
**Pull request by cheratomo**

### CODE REVIEW

**Comment by cheratomo** [comment]
add members A-D

File: ra-admin/members.json [hunk 1]
```diff
@@ -504,6 +504,16 @@
...
+    },
+    {
+      "name": "Ajent Vee",
+      "title": "",
+      "type": "Pokemon",
+      "description": "Vee is a lesbian streamer and artist who primarily plays Pokemon related games. A fan of achievements and hunting for the rarest Pokemon out there, she welcomes memes and good vibes into her community, making it a safe and inclusive space for all.",
+      "priority": 1,
+      "twitch": "AjentVee",
+      "twitter": "AjentVee",
+      "image": "ajentvee.png"
```
Comment (meastoso) [inline ra-admin/members.json:516]: Image references are case sensitive. Looks like the file you added is "AjentVee.png" so just align the capitalization here as well.

**Review by meastoso — COMMENTED** [review]
The file name added is: "Copy of AlleeMac_RainbowArcade_Headshot.png". This format with spaces will cause issues when the page loads when finding the file. Please use more simple filenames with no special characters beyond hyphens or underscores.

**Review by meastoso — CHANGES_REQUESTED** [review]
Please use simple filenames for photos, this won't get picked-up by the website properly.

**Comment by cheratomo** [comment]
Updated - photo files fixed!

**Comment by cheratomo** [comment]
added the rest of the members to photos/members.json; have some concern about whether non-vanity youtube channel links will work in this format

**Pull request merged**